Documentation/configuration/verify-image-signatures.rst

```diff
@@ -0,0 +1,36 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _verify_image_signatures:
+
+**************************
+Verifying Image Signatures
+**************************
+
+Prerequisites
+=============
+
+You will need to `install cosign`_.
+
+.. _`install cosign`: https://docs.sigstore.dev/cosign/installation/
+
+Verify Signed Container Images
+==============================
+
+For a complete list of images that are signed, please refer to `Releases`_.
+
+Let's pick one image from this list and verify its signature using the ``cosign verify`` command:
+
+.. code-block:: shell-session
+
+    $ COSIGN_EXPERIMENTAL=1 cosign verify quay.io/cilium/cilium:v1.x.x
```

Comment:
Hi! Just chiming in. Great work! I saw Dan tweet about it earlier.

Just FYI: this verification checks if there is *ANY* valid signature on the image, but not who or what signed it to determine it's trust. This means that anyone (with write access to the container) could have signed it in order to produce a positive verification, but *not* necssarily the cilium repository or maintainers.

See https://github.com/sigstore/cosign/issues/2056

I would recommend adding in a `--cert-identity .github/workflows/build-images-hotfixes.yaml` or something similar, that would allow a user to validate that the cilium repository signed the image. You could also sign your images with a service account email that the workflow authenticates to. Feel free to tag me if you're interested in something like that!